common/templates/collection-as-table.njk

```diff
@@ -30,4 +30,24 @@
       }
     }) }}
   {% endif %}
+
+  {% if resultsAsTable.cancelled.rows.length %}
+    {% set html %}
+      <div>
```

Comment:
Do we need this extra `div` or can we just use the `table` element that is output from `govukTable`?